### PR TITLE
Translation hi

### PR DESCRIPTION
**Pull request by gautamajay52**
Hello I have translated the files into hindi.
And you can see that some words are still in english because to get better understanding the app in hindi...that words should not be translated to hindi. Otherwise it will cause problem to man who are using this app in hindi.
Wating for your suggestions, and I am here to help.

Thanks.

### CODE REVIEW

File: brevent/src/main/res/values-hi/strings.xml [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+    <string name="brevent_about_system">System status</string>
+    <string name="brevent_about_system_summary">Standby: %1$s\nStopped state: %2$s\nAuto update: %3$s\nAuto brevent: %4$s\nApp ops: %5$s\nDisable Apps: %6$s</string>
+    <string name="brevent_about_system_summary_mock">Standby: %1$s\n(Read-only mode)</string>
+    <string name="brevent_about_system_supported">समर्थित</string>
+    <string name="brevent_about_system_unsupported">असमर्थित</string>
+    <string name="brevent_about_developer">Developer options</string>
+    <string name="brevent_about_developer_adb">\"USB debugging\" is on</string>
+    <string name="brevent_about_developer_adb_v26" />
+    <string name="brevent_about_translation">Translator</string>
+    <string name="brevent_about_translator">DKOUT, Aviraxp, Li Yufan, TacoTheDank</string>
```
Comment (liudongmiao) [inline brevent/src/main/res/values-hi/strings.xml]: Change to your name

File: brevent/src/main/res/values-hi/strings.xml [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+    <string-array name="brefoils">
+        <item>1 Brefoil</item>
+        <item>2 Brefoil</item>
+        <item>3 Brefoil</item>
+        <item>4 Brefoil</item>
+        <item>5 Brefoil</item>
+    </string-array>
+
+    <string name="version_free">मुक्त</string>
+    <string name="version_free_tip">@string/भुगतान</string>
```
Comment (liudongmiao) [inline brevent/src/main/res/values-hi/strings.xml]: keep @string/payment, or translate if it should be different.

File: brevent/src/main/res/values-hi/strings.xml [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+    <string name="version_unlimited">असीमित</string>
+    <string name="version_unlimited_tip">@string/भुगतान</string>
```
Comment (liudongmiao) [inline brevent/src/main/res/values-hi/strings.xml]: keep @string/payment, or translate if it should be different.

**Comment by gautamajay52** [comment]
Hey added my name and kept the @string/payment

**Comment by gautamajay52** [comment]
Updated again...plz review it.

**Comment by liudongmiao** [comment]
From play store, there is no Brevent users in Hindi.

**Pull request closed without merging**